### ISSUE TITLE
Adding 'last opened' plans to either usage statistics or to the table for plan review 

### ISSUE THREAD

**Issue by magdalenadrafiova** [DMPRoadmap/roadmap#2616]
In our DMPonline drop-in session there was a suggestion that with upgrading the plan review table #2365 it will be nice to also be able to see when was a plan last opened/accessed by an admin. 

It would be interesting to collect the data on this and Sam has suggested that this could feed more interesting statistics to the usage dashboard too.

**Comment by briri** [comment]
@mariapraetzellis and I discovered that the upgrade to the latest version caused many of the plan records' `updated_at` date to be the date of the upgrade 👎 

This also got us thinking about the fact that if I go in and add a contributor to my plan or answer a question that the `updated_at` date on the `plans` table does not change and so the tables that display plans 'update' column would technically be inaccurate from the user's point of view

I think we should also consider adding a `changed_date` to the table (and possibly others) to track when a plan has changed.

**Comment by xsrust** [comment]
I've hit that issue with rake:tasks before and found that `object.save(touch: false)` will prevent the timestamp changes (not sure if it's worth going back and changing that task though)

**Comment by briri** [comment]
Yeah, we should probably update those Rake tasks to prevent others from encountering this issue
